### PR TITLE
Add link from footnote back up to citation

### PR DESCRIPTION
**Pull request by rememberlenny**
# Reference 

#704

### CODE REVIEW

File: ui/components/content-renderers/footnote-citation-in-table.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ import Link from '../link';
 export default function FootnoteCitationInTable({ content }) {
   const href = `#${content.footnote_node.identifier}-table`;
   return (
-    <sup className="footnote-citation-in-table">
+    <sup className="footnote-citation-in-table" id={`citation-${content.text}`}>
```
Comment (cmc333333) [inline ui/components/content-renderers/footnote-citation-in-table.js]: The text of a footnote citation and the marker of the footnote it refers to might be slightly different. Can we use `content.footnote_node.identifier` in both, similar to line 7?

File: ui/components/node-renderers/footnote.js [hunk 2]
```diff
@@ -9,7 +9,9 @@ export default function Footnote({ docNode }) {
     <span className="clearfix node-footnote" id={docNode.identifier}>
       <span className="node-footnote-content">
         <span className="citation-marker">
-          { docNode.marker }
+          <a href={`#citation-${docNode.marker}`}>
```
Comment (cmc333333) [inline ui/components/node-renderers/footnote.js]: Let's use `Link` for consistency. It should handle anchors correctly.

File: ui/css/module/_footnotes.scss [hunk 3]
```diff
@@ -128,6 +128,10 @@ $footnote-vertical-margin: 24px;
     line-height: 1rem; // like font-text, except adjusting for bold
   }
 
+  a {
+    color: $color-black;
+  }
```
Comment (cmc333333) [inline ui/css/module/_footnotes.scss]: Do we want this to apply to _all_ anchors at the bottom of the page, or just the markers? Designs for external links are blue, for example.

File: ui/components/content-renderers/footnote-citation-in-table.js (hunk 1 above)
Comment (rememberlenny) [reply ui/components/content-renderers/footnote-citation-in-table.js]: Definitely agree on this. Updated.

File: ui/components/node-renderers/footnote.js (hunk 2 above)
Comment (rememberlenny) [reply ui/components/node-renderers/footnote.js]: Added!

File: ui/css/module/_footnotes.scss (hunk 3 above)
Comment (rememberlenny) [reply ui/css/module/_footnotes.scss]: Not at all. Good catch.

**Comment by rememberlenny** [comment]
@cmc333333 Updated with changes.

**Pull request merged**